registries/adapters/galaxy_adapter.go

```diff
@@ -180,8 +192,9 @@ func (r GalaxyAdapter) getNextImages(ctx context.Context,
 	// Keep getting the images
 	if iResp.Next != "" {
 		log.Debugf("getting next page of results - %v", iResp.Next)
+		searchURL := fmt.Sprintf(galaxyAPIURL, r.Config.URL.String(), iResp.Next)
 		// Fan out calls to get the next images.
-		go r.getNextImages(ctx, fmt.Sprintf("%v%v", galaxyAPIURL, iResp.Next), ch, cancelFunc)
+		go r.getNextImages(ctx, fmt.Sprintf("%v%v", searchURL, iResp.Next), ch, cancelFunc)
```

Comment:
Looking at the response body for a paged request with a next (https://galaxy-qa.ansible.com/api/v1/content/?content_type__name=apb&page_size=2), this is going to request a bad URL:

`"next": "/content/?content_type__name=apb&page=2&page_size=2",`

The `searchURL` here already has the path in it, so you're going to end up with something like:

`https://galaxy-qa.ansible.com/api/v1/content/?content_type__name=apb/content/?content_type__name=apb&page=2&page_size=2`

I'm thinking these fan out calls should do something like `fmt.Sprintf("%v%v", r.Config.URL, iResp.Next)`, since the next value contains the full next path, so we just need to tack the host onto the front of it.

Reply:
```
searchURL := fmt.Sprintf(galaxyAPIURL, r.Config.URL.String(), iResp.Next)
```

My belief is that I'm constructing the `searchURL` correctly, 
That should take the host `https://galaxy-qa.ansible.com` + `/api/v1` + `/content/?content_type__name=apb&page=2&page_size=2`. That looks correct to me.

I think my mistake was simply not using the `searchURL`:

```
go r.getNextImages(ctx, searchURL, ch, cancelFunc)
```